### ISSUE TITLE
new macOS problems with mojave-base

### ISSUE THREAD

**Issue by duckinator** [cirruslabs/cirrus-ci-docs#499]
# Expected Behavior

Adding an unrelated YAML file should not have caused CI to fail.

# Real Behavior

1. I opened this PR, which adds `.github/FUNDING.yml`: https://github.com/duckinator/emanate/pull/72
2. This resulted in this task failing: https://cirrus-ci.com/task/6502847826362368

It appears the `.pkg` file that's _included in the OS by default_ is not signed properly anymore, or something became more strict about package signing.

Logs:

```
sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /
installer: Package name is macOS_SDK_headers_for_macOS_10.14
installer: Certificate used to sign package is not trusted. Use -allowUntrusted to override.
```

# Related Info

- [ ] Website issue
  - Link to page:
- [x] Task issue
  - OS: macOS
  - Task name: macOS PYTHON:3.6.8, macOS PYTHON:3.7.2
  - Script/cache name (if applies):

**Comment by duckinator** [comment]
The logs say adding `-allowUntrusted` can override it, but I'm hesitant to do that without finding out what caused the change because it's not been needed for the 7 months we've been using Cirrus CI for Emanate.

**Comment by fkorotkov** [comment]
I see similar reports for [GitHub Actions](https://github.com/python-pillow/Pillow/issues/4166) and [Travis CI](https://issues.apache.org/jira/browse/ARROW-6987). Not sure what happened there but `mojave-base` VM hasn't changed in a while. 🤔 Will see if someone in the issues above will figure it out before investigating.

**Comment by duckinator** [comment]
@fkorotkov could the certificate used to sign the package have expired? if so, would rebuilding the VM possibly help?

**Comment by fkorotkov** [comment]
I wasn't sure but seems this is the issue: https://eclecticlight.co/2019/10/18/beware-apple-security-certificates-after-24-october-they-may-have-expired/. Will rebuilds the VM soonish (unfortunately it's not as trivial as Docker 😞).

**Comment by duckinator** [comment]
@fkorotkov any luck with the new macOS VM images? Or, alternatively, any ideas for a better way to get Python installed for macOS tasks, since that's what I was using it for?

**Comment by RDIL** [comment]
Homebrew?

**Comment by fkorotkov** [comment]
The image was rebuild since that time. Do you still see the issue?

**Comment by fkorotkov** [comment]
Yeah, is was rebuild on 11/11.

**Comment by duckinator** [comment]
@fkorotkov I was able to reproduce it right before I made my last comment.

On the upside, switching to just installing Python directly via Homebrew &mdash; instead of installing Pyenv via Homebrew then installing Python using Pyenv &mdash; works. (With the caveat that I can only test on the latest Python 3 version that Homebrew provides, instead of specific 3.x versions, but that should be okay in this case.)

So the issue's still there, but I basically worked around it by avoiding the thing that required using `installer` at all. :woman_shrugging:
